From 39c54ebce039cf8a2b1f33425bf75fea32cc5edf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bernhard=20Rosenkr=C3=A4nzer?= <bero@lindev.ch>
Date: Fri, 24 Jun 2022 20:39:03 +0200
Subject: [PATCH] Fix unterminated macros in rpmuncompress

Fix unterminated macros being called when rpmuncompress
tries to expand an lrzip, 7zip or zstd compressed file
---
 tools/rpmuncompress.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/rpmuncompress.c b/tools/rpmuncompress.c
index 55aac4c44d..bd4146d540 100644
--- a/tools/rpmuncompress.c
+++ b/tools/rpmuncompress.c
@@ -41,9 +41,9 @@ struct archiveType_s {
     { COMPRESSED_LZMA,	0,	"%{__xz}",	"-dc",		"" },
     { COMPRESSED_XZ,	0,	"%{__xz}",	"-dc",		"" },
     { COMPRESSED_LZIP,	0,	"%{__lzip}",	"-dc",		"" },
-    { COMPRESSED_LRZIP,	0,	"%{__lrzip",	"-dqo-",	"" },
-    { COMPRESSED_7ZIP,	1,	"%{__7zip",	"x",		"" },
-    { COMPRESSED_ZSTD,	0,	"%{__zstd",	"-dc",		"" },
+    { COMPRESSED_LRZIP,	0,	"%{__lrzip}",	"-dqo-",	"" },
+    { COMPRESSED_7ZIP,	1,	"%{__7zip}",	"x",		"" },
+    { COMPRESSED_ZSTD,	0,	"%{__zstd}",	"-dc",		"" },
     { COMPRESSED_GEM,	1,	"%{__gem}",	"unpack",	"" },
     { -1,		0,	NULL,		NULL,		NULL },
 };
